cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -96,7 +98,7 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
       text =
         """include::list-constraints-table-columns.asciidoc[]
           |
-          |The old built-in procedures for listing constraints, such as `db.constraints`, work as before and are not affected by the
+          |Note that the old deprecated built-in procedures for listing constraints, such as `db.constraints`, work as before and are not affected by the
```

Comment:
```suggestion
          |Note that the deprecated built-in procedures for listing constraints, such as `db.constraints`, work as before and are not affected by the
```
This should be an admonition, for consistency with its equivalent in the `index` section. I'll be happy to fix in a separate PR.

Reply:
The only reason its not is due to me not succeeding in doing that using this test setup. Feel free to fix in a separate PR (might be interested in looking at it to see your solution 😄)